docs/http/traffic-policy/index.mdx

```diff
@@ -23,7 +23,7 @@ import TrafficPolicyWarning from "/shared/warnings/traffic-policy-preview.mdx";
 ## Overview
 
 This module allows you to assign a policy to your endpoints containing a series
-of inbound and outbound rules that can be used to influence and control traffic
+of `on_http_request` and `on_http_response` rules that can be used to influence and control traffic
 to and from your upstream service.
```

Comment:
Hmm might need some rewrite in here because HTTP endpoints also support the `on_tcp_connect` phase